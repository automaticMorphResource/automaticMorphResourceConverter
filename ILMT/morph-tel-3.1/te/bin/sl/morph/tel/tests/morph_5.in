1	BUmiki	unk	
1	cAlA	unk	
1	eVwwu	unk	
1	nuMci	unk	
1	kUdA	unk	
1	ixi	unk	
1	kanipiswuMxi	unk
1	ani	unk	
1	ceVbuwuMtAru	unk	
1	.	unk	
