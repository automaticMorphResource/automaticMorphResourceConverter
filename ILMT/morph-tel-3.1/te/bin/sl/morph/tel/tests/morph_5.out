1	BUmiki	unk	<fs af='BUmi,n,,sg,,,ki,ki'>
2	cAlA	unk	<fs af='cAlA,avy,,,,,0,0_avy'>|<fs af='cAlu,v,any,any,any,,i_A,i_A'>|<fs af='cAlu,avy,,,,,A,A_avy'>
3	eVwwu	unk	<fs af='eVwwu,adj,,,,,0,0_adj'>|<fs af='eVwwu,n,,sg,,d,0,0'>|<fs af='eVwwu,n,,sg,,o,ti,ti'>|<fs af='eVwwu,v,,sg,2,,AjFArWa,AjFArWa'>
4	nuMci	unk	<fs af='nuMci,avy,,,,,0,0_avy'>
5	kUdA	unk	<fs af='kUdA,avy,,,,,0,0_avy'>|<fs af='kUdu,v,any,any,any,,i_A,i_A'>|<fs af='kUdu,n,,sg,,,0,0_A'>
6	ixi	unk	<fs af='ixi,pn,fn,sg,3,d,0,0'>
7	kanipiswuMxi	unk	<fs af='kanipiMcu,v,fn,sg,3,,wA,wA'>
8	ani	unk	<fs af='ani,avy,,,,,0,0_avy'>|<fs af='ani,avy,,,,,0,0_avy'>|<fs af='anu,v,any,any,any,,i,i'>
9	ceVbuwuMtAru	unk	<fs af='ceVppu,v,fm,pl,3,,wU_uMdu_wA,wU_uMdu_wA'>
10	.	unk	<fs af='&dot;,punc,,,,,,'>
